### DISCUSSION TITLE
Adding an interactivity step to SD onboarding flow

### DISCUSSION THREAD

**Discussion by wlstylr** [SecretDecks/Documentation#41] [Ideas]
Hello all,
@DanM3rcurius and I were discussing the merits of incorporating [this game ](https://ncase.me/trust/) into the SD onboarding flow.  I think we both agree that it is nice interactive way to introduce folks to trust and game theory concepts, but we are unsure of the best way to incorporate.  Should it be mandatory or optional?  Should we have a test associated with it?  Any other ideas on how we can use this as a launching point in designing a gamified learning experience?

**Comment by stephen-rowan** [comment]
A charming introduction. I would recommend this as an on-boarding exercise - it would also be good to discuss this as a community and to workshop at some point. It could become mandatory through habit of reference perhaps ? A must-see ?

It also can double as a focus of discussion in how to build trust in communities - so is related to a Code of Conduct (something I am working on at present). For example Catalyst Community Guidelines cite "Assume good intent" - a win/win strategy and while this is a good approach to life it is also vulnerable to bad actors ("The cheater"). Which is why a Code of Conduct always has a remediation or enforcement section that resets or regulates the game. 

It also usefully highlights risks that cause breakdown in trust - in particular : 1) mistakes, 2) scaling/complexity 3) miscommunication. 

- **Mistakes** - what mistakes do we forgive ? Perhaps this is related to a person's track record or reputation ? But it can also be related to a person's capability - a sign that they need help in learning. Or it can be an excuse for bad faith.

- **Scaling/complexity** - it is harder to maintain trust at scale, which may have implications for community building. Small clusters may be better (have more variety) than massive swarms (which optimize on a single outcome).

- **Miscommunication** - whatever the scale, lack of transparency will cause zero-sum behavior. Whenever a player feels they lack knowledge or access to information is blocked to them. At a small scale this can arise through the silo effect and at a mass scale through complexity (too many divergent sources of information).

An interesting sequel might focus on the difficulty of maintaining non-zero sum games - particularly in multi-player scenarios and in continuous situations.

**Reply by wlstylr** [reply]
Thank you for additional input and suggestions.  I agree that it should be an on-boarding exercise, and that community discussions and workshops would be a great extension.  I think the easiest and quickest way to start incorporating would be to include it in the Discord community welcome section that @DanM3rcurius has been working on.  

Great breakdown regarding mistakes, scaling/complexity, and miscommunication, and will try to add a bit below:

- **Mistakes** - Being able to properly understand the reason behind the mistake is of the utmost importance.  I think we are taking the right steps in creating the MP and Charter before scaling the community.  When mistakes are made perhaps we should have a learning path available.  We should be forgiving and offer solutions, but should not be naive and let mistakes multiply to the point of being corrosive.
- **Scaling/complexity** - I agree that small clusters with more variety will be more optimal than massive swarms optimizing a single outcome.  Processes that prioritize transparency will be important in mitigating swarm silos. 
- **Miscommunication** - 100% agree that lack of transparency breeds zero-sum behavior.  Something else to be aware of is intentional miscommunication through the dishonest framing of information. 

I like the sequel idea.  Regardless if the original creator decides to design a follow-up, I think it would be in the ACO's best interest to host regular workshops exploring topics around maintaining non-zero sum games.

**Reply by stephen-rowan** [reply]
@wlstylr and @DanM3rcurius - I posted on a link to this game on the Catalyst #game-theory channel here ( https://discordapp.com/channels/756943420660121600/828586061303250964/852341280046317578) - let me know if you want me to credit you as a source.

**Reply by wlstylr** [reply]
No worries, I just shared it here and didn't give credit to the source I found it from.  Thanks for sharing on Catalyst Discord :)

**Comment by DanM3rcurius** [comment]
hehe all good, no need to give credit; you sharing it is credit enough(:
I show this to everyone who I'm initiating about crypto since a couple of years. It's just genius and should be spread far and wide!

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by DanM3rcurius** [comment]
can we integrate also the voting process in action into the onboarding? or is that too much? @wlstylr ?

**Reply by wlstylr** [reply]
Do you mean instructions on how to vote in the onboarding flow?  If so, I think that is too much info for a person that is new to Secret Decks.  I think how to vote should happen after progressing to Core.  However, I think this will evolve as we better define our voting system in a way that is easy to absorb.

**Reply by DanM3rcurius** [reply]
ok agreed!
